src/python_testing/TC_AccessChecker.py

```diff
@@ -61,10 +95,19 @@ def checkable_attributes(cluster_id, cluster, xml_cluster) -> list[uint]:
 
     def known_cluster_attribute(attribute_id) -> bool:
         ''' Returns true if this is a non-manufacturer specific attribute that has information in the XML and has python codegen data'''
-        return attribute_id <= 0xFFFF and attribute_id in xml_cluster.attributes and attribute_id in Clusters.ClusterObjects.ALL_ATTRIBUTES[cluster_id]
+        return is_standard_attribute_id(attribute_id) and attribute_id in xml_cluster.attributes and attribute_id in Clusters.ClusterObjects.ALL_ATTRIBUTES[cluster_id]
     return [x for x in all_attrs if known_cluster_attribute(x)]
 
 
+def checkable_commands(cluster_id, cluster, xml_cluster) -> list[uint]:
+    all_cmds = cluster[GlobalAttributeIds.ACCEPTED_COMMAND_LIST_ID]
+
+    def known_cluster_cmds(command_id) -> bool:
```

Comment:
nit: `def is_known_cluster_cmd(command_id: int) -> bool` as it's a single predicate and does not return a,ist.